rskj-core/src/test/java/co/rsk/peg/federation/FederationStorageProviderImplTests.java

```diff
@@ -892,6 +892,60 @@ void testSaveOldFederation(
         );
     }
 
+    @Test
+    void savePendingFederation_postWasabi_shouldBeSavedInStorageWithFormatVersion() {
```

Comment:
```suggestion
    void savePendingFederation_whenWasabiIsActive_shouldBeSavedInStorageWithFormatVersion() {
```

Reply:
@julia-zack recommended me a while ago the following format for test names, and I consider it makes easier to understand what the test does:

`methodName_when<testCase>_shouldReturn/shouldSave`

In this case, we might put something like:

`save_whenWasabiIsActive_shouldSavePendingFederationInStorageWithFormatVersion()
`

WDYT, guys?
C.C.: @jeremy-then @marcos-iov @nathanieliov @apancorb @julia-zack

Reply:
This was voted for in my polly. Let's take the conversation there if needed.